### ISSUE TITLE
Accessibility > Names and Labels 

### ISSUE THREAD

**Issue by pocc** [pocc/tshark.dev#16]
* [ ] __Form elements do not have associated labels__
Labels ensure that form controls are announced properly by assistive technologies, like screen readers. [Learn more](https://dequeuniversity.com/rules/axe/3.3/label).
    ```
    Failing Elements
    input#search-by
    ```
